app/src/atoms/SoftwareKeyboard/CustomKeyboard/index.css

```
@@ -3,7 +3,7 @@
 .simple-keyboard.oddTheme1.hg-theme-default {
   width: 100%;
   height: 100%;
-  background-color: #d0d0d0;
+  background-color: #cbcccc;
```

Sorry, this is my fault, could you add note?

```suggestion
  background-color: #cbcccc; /* grey35 */
```